app/models/concerns/decidim/kids/user_override.rb

```diff
@@ -25,6 +25,13 @@ def minor?
         def tutor?
           minor_accounts.exists?
         end
+
+        def minor_age
+          return unless minor?
+          return if minor_data_birthday.blank?
+
+          Time.zone.today.year - minor_data_birthday.year
```

Comment:
If the minor birthday is in December, we will get one more year than expected in the other months.

I've found [here](https://medium.com/@craigsheen/calculating-age-in-rails-9bb661f11303) an easy way to do it:

```suggestion
          ((Time.zone.now - minor_data_birthday.to_time) / 1.year.seconds).floor
```

Reply:
Good point, added a spec too to check the edge case